pkg/pluginmanager/manager.go

```diff
@@ -200,16 +200,18 @@ func DiscoverPluginGroups() ([]*discovery.DiscoveredPluginGroups, error) {
 }
 
 // getPreReleasePluginDiscovery
-// For our pre-releases we use an environment variable to point to the
-// repository of plugins.  This is because the configuration
-// cfg.ClientOptions.CLI.DiscoverySources
+// For pre-releases CLI points to a default staging central plugin discovery image
+// from where the CLI will discover plugins.
+// For pre-releases, CLI also allows default discovery image to be overriden by using
```

Comment:
As suggested by the linter:
```suggestion
// For pre-releases, CLI also allows default discovery image to be overridden by using
```